.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,61 @@
+name: Benchmarks
+
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
```

Comment:
I think that we should remove from `pull_requests` and try this like cd_dev.yaml, not required for merge but nice to have monitored